### ISSUE TITLE
New bone_eqep2b.dts has errors.

### ISSUE THREAD

**Issue by VoltVisionFrenchy** [teknoman117/beaglebot#4]
I think the new bone_eqep2b.dts that was added has incorrect comments.
The author updated the P8 pin numbers, but not the GPIO numbers, therefore they are not correct.  Thx!  ( I would try to fix it, but I am new to Github and I dont know how)...

**Comment by pdp7** [comment]
I think they are the same GPIO numbers but brought out to different pins on the header?
I've been using eqep2 with alternate header pins P8_11 & P8_12 after seeing this post from BeagleBoard mailing list:
https://groups.google.com/forum/#!msg/beagleboard/0ewLcdUtA_E/sVEoDGkL9FcJ
And this device tree config for eQEP2:
https://17376924468624948387.googlegroups.com/attach/57f40b690c2851b1/LoadeQEP.txt?part=0.1&view=1&vt=ANaJVrGPwHIS2WLiOv6ixKKbzR46eHqySZLgFoyqQ_d_WnwWpd6WHjAYJi1DUftPPemLjzWziNVQ5BXCSQEDe6COpc_1ZPRU5ZAgS0AyU5ycCZ0_vLyWF3g

**Comment by jadedanemone** [comment]
The comments are definitely wrong, even in my original postings, even though they're all a little different. I'd fix it, but my computer just kicked the bucket. Take a look at the BBB SRM. All of the eqep2b pins are on GPIO1.

**Comment by pdp7** [comment]
Right, I chatted with VoltVisionSteve on freenode #beagle-gsoc and I believe this should be the correct GPIOx_y numbers:

<pre>
                        0x038 0x24  /* P8_16 = GPIO1_14 = EQEP2_index, MODE4 */
                        0x03C 0x24  /* P8_15 = GPIO1_15 = EQEP2_strobe, MODE4 */
                        0x030 0x34  /* P8_12 = GPIO1_12 = EQEP2A_in, MODE4 */
                        0x034 0x34  /* P8_11 = GPIO1_13 = EQEP2B_in, MODE4 */
</pre>

**Comment by pdp7** [comment]
Here is the TRM page that I looked at the pin table from:
https://plus.google.com/photos/117542001281850354871/albums/6028753166053143281/6029361863734888530?banner=pwa&pid=6029361863734888530&oid=117542001281850354871

**Comment by VoltVisionFrenchy** [comment]
Drew,
Thanks for chiming in!  Actually that Table you reference is from the SRM, not the TRM.... But hopefully they all match!  I originally found the conflict by looking at the same SRM (from the BBB main wiki page).   Thx again!

## 

Respectfully,
Steve French
President, Volt Vision

800.664.7256.office
814.730.0003.cell
www.voltvision.com
